### DISCUSSION TITLE
[Mini-évent] Croiser Gruffo qui propose au joueur de faire un échange de potion ou d'en racheter une

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#874] [Nouvelles idées]
Au vu du nombre de potions qu'on peut drop et au final qui finissent dans les éviers, je pensais faire intervenir Gruffo par le biais d'un mini-évent où l'on peut choisir soit : 

  -d'échanger une potion contre une autre potion random. 
  
  -de vendre une unique potion (choisis ou non) en notre possession à Gruffo. (avec prix de vente majoré, équivalent et/ou minoré).

**Comment by LeHoublonPensant** [comment]
> 

Oui voilà parce que c'est pas bien de gaspiller 😂
